### ISSUE TITLE
Can I get the Hexbin layer's boundary data? 

### ISSUE THREAD

**Issue by YinZhaoJie** [keplergl/kepler.gl#1095]
# Could Hexbin layer data can be download in any formart (geojson,csv,etc..)?

**Comment by rogerganga** [comment]
Hi do you mean adding hexbin layer into kepler or into GIS? 

Where do you want the hexbin layer to be downloaded from? 

Possible to add any image?

**Comment by YinZhaoJie** [comment]
> Do you mean adding hexbin layer into kepler or into GIS?
> 
> Where do you want the hexbin layer to be downloaded from?
> 
> Possible to add any image?

Hi rogerganga, I want  to download the Hexbin layer vector data (maybe in pixel xys )  from kepler.gl, and   it would be great helpful that   it could be downloaded in geographic coordinates.
Beacuse I can get where my hot place is  in kepler.gl 's Hexbin layer ,and  I  need the boundary of each  hexagons to get (ST_intersects)  other data in my PostgreSQL.

**Comment by rogerganga** [comment]
Hi @YinZhaoJie ! If you mean exporting the below hexbin layer, then I am not sure if there is a way to do it. A possible solution that I can think of is it pragmatically convert your coordinates into a hexbin structure and export it as shapefile or geojson and import in your GIS software. I am not sure if you can directly export the below structure as a shapefile.  

![image](https://user-images.githubusercontent.com/34466964/80855232-0702b600-8c72-11ea-983f-f9495819ea42.png)

**Comment by heshan0131** [comment]
export directive layer data is not possible right now, however, it is a necessary feature to add

**Comment by YinZhaoJie** [comment]
> export directive layer data is not possible right now, however, it is a necessary feature to add

Yes, it will be a great tool  if it can export directive layer data. Best wishes to kepler.gl .

**Comment by Geschwindigkeitsbegrenzung** [comment]
This would be of great benefit. Currently seeking a work around to display H3 aggregations outside of Kepler.
